keycloak/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi9 AS ubi-micro-build
+RUN mkdir -p /mnt/rootfs
+RUN dnf install --installroot /mnt/rootfs curl --releasever 9 --setopt install_weak_deps=false --nodocs -y && \
+    dnf --installroot /mnt/rootfs clean all && \
+    rpm --root /mnt/rootfs -e --nodeps setup
+
+FROM quay.io/keycloak/keycloak:latest
+
+COPY --from=ubi-micro-build /mnt/rootfs /
+
+# Enable health and metrics support
+ENV KC_HEALTH_ENABLED=true
+
+# change these values to point to a running postgres instance
+ENV KC_DB=postgres
+ENV KC_DB_URL=<DBURL>
```

Comment:
They are needed because we need to declare that variables are exposed and required to build and run this image, otherwise they will not be replaced.

I tried removing them just in case and the build failed.
```
gdi-userportal-ckan-docker-keycloak-1  | 2023-12-01 09:59:12,279 WARN  [io.quarkus.agroal.runtime.DataSources] (main) Datasource <default> enables XA but transaction recovery is not enabled. Please enable transaction recovery by setting quarkus.transaction-manager.enable-recovery=true, otherwise data may be lost if the application is terminated abruptly
gdi-userportal-ckan-docker-keycloak-1  | 2023-12-01 09:59:12,342 WARN  [io.agroal.pool] (agroal-11) Datasource '<default>': URL format error; must be "jdbc:h2:{ {.|mem:}[name] | [file:]fileName | {tcp|ssl}:[//]server[:port][,server2[:port]]/name }[;key=value...]" but is "jdbc:postgresql://db/keycloakdb" [90046-224]
gdi-userportal-ckan-docker-keycloak-1  | 2023-12-01 09:59:12,343 WARN  [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator] (JPA Startup Thread) HHH000342: Could not obtain connection to query metadata: java.lang.NullPointerException: Cannot invoke "org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(java.sql.SQLException, String)" because the return value of "org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.sqlExceptionHelper()" is null
```